ocp4/profiles/cis.profile

```diff
@@ -169,7 +169,13 @@ selections:
     - kubelet_configure_tls_key
     # Like kubelet_disable_readonly_port but check for .apiServerArguments["kubelet-client-key"]
   # 4.2.13 Ensure that the Kubelet only makes use of Strong Cryptographic Ciphers
+    - kubelet_configure_tls_cipher_suites
```

Comment:
Since this is a node-level check it should be in the cis-node.profile instead

Reply:
@mrogers950 thank you, I changed target profile from `cis` to `cis-node`.
Now the check looks fine (no error happened in the CI test).  Are current description of `rule.yml` (comment and logic), test and remediation script OK to be merged?